draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -166,6 +160,11 @@ a RESET_STREAM_AT frame. Similary, the Reliable Size of the RESET_STREAM_AT
 frame does not prevent a QUIC stack from delivering data beyond the specified
 offset to the receiving application.
 
+Note that a Reliable Size value of zero is valid. A RESET_STREAM_AT frame with
+this value is logically equivalent to a RESET_STREAM frame ({{Section 3.2 of
+RFC9000}}). A sender MAY use either RESET_STREAM or RESET_STREAM_AT with a
+Reliable Size of zero to avoid stream data delivery.
```

Comment:
Technically, you don't avoid stream data delivery, you just don't commit.

Reply:
Split the difference and "Avoid committing "? 😃